todoapp/app.py

```diff
@@ -40,13 +40,13 @@ def create_app():
     app.config.from_object('todoapp.flask_config.Config')
 
     file_path = find_dotenv('.env')
-    load_dotenv(file_path, override=True)
+    load_dotenv(file_path, override=False)
```

Comment:
This reduces the risk that your tests pick up live config; but removing these lines entirely should do so further